status/format.go

```diff
@@ -196,3 +207,18 @@ func FormatFootnotes(taskRuns []*helpers.TaskRun) (string, error) {
 	}
 	return strings.Join(footnotes, "\n"), nil
 }
+
+// FormatPipelineURL accepts a name of application, pipelinerun, namespace and returns a complete pipelineURL.
+func FormatPipelineURL(pipelinerun string, namespace string) string {
+	console_url := os.Getenv("CONSOLE_URL")
+	if console_url == "" {
+		return "CONSOLE_URL_NOT_AVAILABLE"
+	}
+	buf := bytes.Buffer{}
+	data := SummaryTemplateData{PipelineRunName: pipelinerun, Namespace: namespace}
+	t := template.Must(template.New("").Parse(console_url))
+	if err := t.Execute(&buf, data); err != nil {
+		log.Fatal(err)
```

Comment:
do we really want to panic and kill whole service here?
I'd prefer something that would only fail to create report